orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/providers/aws/ModifyAwsScalingProcessStage.groovy

```diff
@@ -101,7 +101,7 @@ class ModifyAwsScalingProcessStage extends TargetServerGroupLinearStageSupport i
         throw new IllegalStateException("No server group found (serverGroupName: ${stageData.region}:${stageData.serverGroupName})")
       }
 
-      def suspendedProcesses = getSuspendedProcesses(targetServerGroup.get())
+      def suspendedProcesses = targetServerGroup.get().getSuspendedProcesses()
```

Comment:
Risk of an NPE here.

Reply:
how so? There is a `targetServerGroup.present` check above